packages/api/lib/reconciliationReport.js

```diff
@@ -263,7 +263,7 @@ function compareEsGranuleAndApiGranule(esGranule, apiGranule) {
   if (granulesAreEqual === false) return granulesAreEqual;
 
   const esGranulesHasFiles = esGranule.files !== undefined;
-  const apiGranuleHasFiles = apiGranule.files !== undefined;
+  const apiGranuleHasFiles = apiGranule.files.length !== 0;
```

Comment:
Should `esGranulesHasFiles` be updated to check for an empty array instead of undefined, as well?